vscode/quint-vscode/server/src/complete.ts

```diff
@@ -132,20 +167,20 @@ class DeclOfNameFinder implements IRVisitor {
  *
  * @returns Array of IDs that declare `name`.
  */
-function findDeclByNameInsideScope(name: string, scopeId: bigint, modules: QuintModule[]): bigint[] {
-  const module = modules.find(module => module.id == scopeId)
+function findDeclByNameInsideScope(name: string, scopeId: string, modules: QuintModule[]): string[] {
+  const module = modules.find(module => module.id.toString() == scopeId)
   if (module) {
-    // scope is a module; return all variables with name `name`
-    return module.declarations.filter(decl => decl.kind == 'var' && decl.name == name).map(decl => decl.id)
+    // scope is a module return all variables with name `name`
+    return module.declarations.filter(decl => decl.kind == 'var' && decl.name == name).map(decl => decl.id.toString()) // Convert BigInt to string
   }
-  const expr = findExpressionWithId(modules, scopeId)
+  const expr = findExpressionWithId(modules, BigInt(scopeId))
   if (expr) {
-    // scope is an expression; recursively search it for declarations of the name `name`
+    // scope is an expression recursively search it for declarations of the name `name`
```

Comment:
We need some punctuation on this sentences - I'm guessing you removed all `;` from the file and got these accidentally removed
```suggestion
    // scope is an expression; recursively search it for declarations of the name `name`
```